step-certificates/README.md

```diff
@@ -60,6 +60,7 @@ chart and their default values.
 | `ca.db.size`                | Persistent volume size                                                            | `10Gi`                        |
 | `ca.runAsRoot`              | Run the CA as root.                                                               | `false`                       |
 | `ca.bootstrap.postInitHook` | Extra script snippet to run after `step ca init` has completed.                   | `""`                          |
+| `ca.bootstrap.rootCA.secret`| Name of the custom root CA secret (k8s tls secret) to be used.                    | `""`                          |
```

Comment:
Hello @maraino . Thanks for your comments. Agree with parametrizing cert and key file names, but disagree with using configmap. Correct is to use kubernetes secret, which can be still used and meet your requirements as well.
I would propose to use generic k8s secret  as follows:
`kubectl create secret generic my-root-ca --from-file=certs/root_ca.crt --from-file=certs/root.key `

and passing to smallstep helmchart `rootCA.secret.name`, `rootCA.cert` `rootCA.key`.
where `rootCA.secret.name: my-root-ca`, `rootCA.cert: root_ca.crt` and `rootCA.key: root.key`

Do you agree?

Reply:
The certificate itself is not really secret, but what I wanted address here is to be able to mount from CSI interfaces, and get the key from for example Vault.

To simplify the integration makes sense to just add one volume instead of two. Both a secret and CSI can have multiple objects.